arches_component_lab/src/arches_component_lab/types.ts

```diff
@@ -24,28 +24,3 @@ export interface CardXNodeXWidget {
         component: string;
     };
 }
-
-export interface Language {
-    code: string;
-    default_direction: "ltr" | "rtl";
-    id: number;
-    isdefault: boolean;
-    name: string;
-    scope: string;
-}
```

Comment:
I think this one would have been nice to leave here.

Reply:
That said I'll just update controlled-lists, but we're going to need to decide where to put types that represent core arches concepts.